firebase-vscode/src/extension.ts

```diff
@@ -1,5 +1,5 @@
 import * as vscode from "vscode";
-import { spawnSync } from 'child_process';
+import { spawnSync } from "child_process";
```

Comment:
Lots of prettier diffs. Maybe we should add prettier lint to the CI to make sure everyone else runs it.

Reply:
++ I'll look into it